devtools/conda-envs/psi4.yaml

```
@@ -47,7 +47,7 @@ dependencies:
   - h5py>=3.6.0
 
   # Optional
-  - openmmforcefields >=0.9.0
+  - openmmforcefields ~=0.14.2
```

(not blocking) ditto
```suggestion
  - openmmforcefields >=0.14.2
```